en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/passing-enduser-attributes-to-the-backend-via-api-gateway.md

```diff
@@ -14,6 +14,56 @@ Use the following format to encode the JWT using Base64URL encoding, by adding t
 encoding = "base64url"
 ```
 
+## Validating backend JWT using JWKS endpoint
```

Comment:
Shall we add a note saying this will be available from update level x ?

Reply:
Added the following description regarding the update level

<img width="905" alt="image" src="https://github.com/wso2/docs-apim/assets/30475839/a328ca0d-2546-4a07-924d-96250102859b">